### ISSUE TITLE
Performance difference while running M4C pretrained model

### ISSUE THREAD

**Issue by aditya5558** [facebookresearch/mmf#661]
## ❓ Questions and Help

I ran the M4C pretrained model following the M4C documentation using the following command:

`mmf_run dataset=textvqa   model=m4c   config=projects/m4c/configs/textvqa/joint_with_stvqa.yaml   env.save_dir=./save/m4c   run_type=val   checkpoint.resume_zoo=m4c.textvqa.with_stvqa training.num_workers=1
`

I am getting a validation accuracy of 0.3882 as compared to 0.4055 which is reported in the paper. Is this difference in performance expected?

**Comment by apsdehal** [comment]
Thanks for reporting this. We will have a look at this. This might be a regression caused by some recent changes.

**Comment by apsdehal** [comment]
@aditya5558 #664 should fix this. You can patch and try.

**Comment by sweta125** [comment]
Where are the pretrained models available as input to checkpoint.resume_zoo , specifically for m4c model? I checked the mmf model zoo documentation but cannot find this.
